Introduction.tex

```diff
@@ -1,7 +1 @@
-
-One facility that conducts spallation experiments is the ISIS Pulsed Muon and Neutron source located in Rutherford Appleton Laboratory, UK. It is a world leader in terms of scientific output and has some number of visitors every year. A similar facility, the European Spallation Source (ESS) in Sweden, is presently under construction. Once completed, the ESS will be the most powerful spallation source in the world. The science that will take place at the ESS will be fundamentally similar to that which takes place at ISIS, so it is anticipated that that these facilities will have similar software needs. As such, the decision was made to have the ESS use much of the software that is presently used at ISIS.
-
-The instruments used to conduct spallation experiments are comprised of various different components that are used either to alter the beam of neutrons or to measure/detect certain phenomena. Such components include neutron detectors, which capture the pulse of energy after going through a target, and disk choppers, which interrupt the beam so that only neutrons of a certain speed may reach the material that we wish to study. Precise knowledge of the properties and whereabouts of these components is vital in order to analyse the results of our experiments reliably. 
-
-The NeXus file format arose out of a desire to describe the components used in these experiments in a way that is ``facility-neutral." An advantage is that this allows for greater openness in scientific research. However, the format is not particularly easy to understand for the unitiated and scientists would would rather get their data and get on with their day. The NeXus Constructor arose out of a collaboration between software developers based at ISIS and the ESS in order to make a tool that allows scientists to easily examine and modify the contents of NeXus files.
+The NeXus file format arose out of a desire to describe the configurations of neutron, muon, and X-ray experiments in a way that is ``facility-neutral." A shared data format has a number of advantages such as reducing the need for conversion tools and fostering cooperation in software development across different facilities. However, the format is not particularly easy to work with for the unitiated. This can be especially frustrating to scientists who would much rather get their data and get on with their day. The NeXus Constructor, a tool being developed by software developers based at ISIS Pulsed Muon and Neutron Source in the UK and the European Spallation Source (ESS) in Sweden, attempts to address this problem by providing an interface for users to easily examine and modify the contents of NeXus files.
```

Comment:
Given the audience I think you need a little more context than this. See the PR I made with changes to the introduction, although what I put there would probably still need editing down a bit. Perhaps using bullet points would help?